src/Infrastructure/Controller/ContactController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Controller;
+
+use Swagger\Annotations as SWG;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+use Twig\Environment;
+
+class ContactController
+{
+    private MailerInterface $mailer;
+    private Environment $twig;
+
+
+    public function __construct(MailerInterface $mailer, Environment $twig)
+    {
+        $this->mailer = $mailer;
+        $this->twig = $twig;
+    }
+
+    /**
+     * @Route(
+     *     "/contact-form",
+     *     name="contact_form",
+     *     methods={"POST"},
+     *     format="json"
+     * )
+     *
+     * @SWG\Response(
+     *     response=200,
+     *     description="received contact form data",
+     * )
+     *
+     */
+
+    public function formDataAction(Request $request): JsonResponse
+    {
+        try {
+            $formData = json_decode($request->getContent());
+        } catch (NotEncodableValueException $notEncodableValueException) {
+            throw new BadRequestHttpException('No valid json', $notEncodableValueException);
+        }
+
+        try {
+            $params = $request->request->all();
+            $file = $request->files->get('file');
+
+            $mailBody = $this->twig->render(
+                '/email/contact_form.html.twig',
+                $params);
+
+            $email = new Email();
+            $email->subject($params['subject']);
+            $email->html($mailBody);
+            $email->to($params['email']);
```

Comment:
```suggestion
            $email->to('contact@print4health.org');
            $email->cc($params['email']);
```
I would make it public that the email was sent to us and then is cc'ed to the sender too.